releases/release-6.5.5.md

```diff
@@ -0,0 +1,85 @@
+---
+title: TiDB 6.5.5 Release Notes
+summary: Learn about the compatibility changes, improvements, and bug fixes in TiDB 6.5.5.
+---
+
+# TiDB 6.5.5 Release Notes
+
+Release date: xx, 2023
+
+TiDB version: 6.5.5
+
+试用链接：[快速体验](https://docs.pingcap.com/zh/tidb/v6.5/quick-start-with-tidb) | [生产部署](https://docs.pingcap.com/zh/tidb/v6.5/production-deployment-using-tiup) | [下载离线包](https://cn.pingcap.com/product-community/?version=v6.5.5#version-list)
+
+## Improvements
+
++ TiDB **tw@qiancai**
+
+    - Support downgrading TiDB from a later version to v6.5 [#45570](https://github.com/pingcap/tidb/issues/45570) @[wjhuang2016](https://github.com/wjhuang2016)
+    - (dup): release-7.3.0.md > # Reliability - Add new optimizer hints, including [`NO_MERGE_JOIN()`], [`NO_INDEX_JOIN()`](/optimizer-hints.md#no_index_joint1_name--tl_name-), [`NO_INDEX_MERGE_JOIN()`](/optimizer-hints.md#no_index_merge_joint1_name--tl_name-), [`NO_HASH_JOIN()`](/optimizer-hints.md#no_hash_joint1_name--tl_name-), and [`NO_INDEX_HASH_JOIN()`](/optimizer-hints.md#no_index_hash_joint1_name--tl_name-) [#45520](https://github.com/pingcap/tidb/issues/45520) @[qw4990](https://github.com/qw4990)
+    - Add request source information related to the coprocessor [#46514](https://github.com/pingcap/tidb/issues/46514) @[you06](https://github.com/you06)
+
++ TiKV **tw@qiancai**
+
+    - Add the backoff mechanism for the PD client in the process of connection retries, which gradually increases retry intervals during error retries to reduce PD pressure [#15428](https://github.com/tikv/tikv/issues/15428) @[nolouch](https://github.com/nolouch)
+    - Avoid holding mutex when writing Titan manifest files to prevent affecting other threads [#15351](https://github.com/tikv/tikv/issues/15351) @[Connor1996](https://github.com/Connor1996)
```

Comment:
```suggestion
```

Reply:
moved this issue to https://github.com/pingcap/docs/pull/14846/commits/9629e0b3da2ab95c0f3e1043aa863c666a9fb620